src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,10 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
+        logger.info("login method was called.Params:login = {}",login);
         //TODO: add corresponding log message about method login was called
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/213448628-1adfeab2-aa94-47d8-a1e9-aa5733ff4998.png)

Reply:
Thanks a lot for remark.